code/script.js

```diff
@@ -223,49 +229,67 @@ const generateBoard = () => {
 }
 
 // Randomly select a person from the characters array and set as the value of the variable called secret
+// return Math.floor(Math.random() * (max - min + 1) + min)
 const setSecret = () => {
-  secret = charactersInPlay[Math.floor(Math.random() * charactersInPlay.length)]
+  secret = charactersInPlay[Math.floor(Math.random() * (charactersInPlay.length -1 - 0 + 1) + 0)];
+  console.log(secret);
```

Comment:
Here I can sense some of Daniel's helpful video lessons :) But what would happen if you remove the -1 - 0 +1 +0?